config/core/configmaps/defaults.yaml

```diff
@@ -54,6 +54,16 @@ data:
     # should also be increased to prevent in-flight requests being disrupted.
     max-revision-timeout-seconds: "600"  # 10 minutes
 
+    # revision-request-start-timeout-seconds contains the default number of
+    # seconds a request will be allowed to stay open while waiting to
+    # receive any bytes from the user's application, if none is specified.
+    revision-request-start-timeout-seconds: "60"
```

Comment:
The name `revision-request-start-timeout-seconds` doesn't match the property `ResponseStartTimeoutSeconds`

Reply:
will change it to `revision-response-start-timeout-seconds`